pkg/codegen/schema/bind.go

```diff
@@ -1123,14 +1129,14 @@ func (t *types) bindObjectTypeDetails(path string, obj *ObjectType, token string
 	}
 
 	properties, propertyMap, propertiesDiags, err := t.bindProperties(path+"/properties", spec.Properties,
```

Comment:
Can we do the validation here after having bound the output properties of the resource? This so that we don't have to pass flags around. Not a blocker though, just a personal preference

Reply:
Yeh I did consider that, can move it there instead of doing flags.